src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -16,7 +19,20 @@
  * and possible threats can be removed first.
  */
 public final class Sanitizer {
-    
+
+    private static PolicyFactory policy = new HtmlPolicyBuilder()
```

Comment:
See if it looks better if `new HtmlPolicyBuilder()` is on the next line as well (indented 8 spaces w.r.t `private`).